.gitignore

```diff
@@ -45,6 +45,7 @@ nosetests.xml
 coverage.xml
 *.cover
 .hypothesis/
+mlruns
```

Comment:
Mind adding a slash here because it's a directory? Otherwise, this looks good to go!

Reply:
Yup, totally. Thanks! I've updated with a new commit